### DISCUSSION TITLE
Question: why don't you use yarn pnp

### DISCUSSION THREAD

**Discussion by original001** [esm-dev/esm.sh#877] [General]
I believe that Yarn PNP could be a better alternative for installing dependencies. It is several times faster than pnpm. You can check it on the pnpm [Benchmarks](https://deepai.org/url). 

people usualy avoid Yarn PnP due to some issues, but they are not relevant for this project. For example, absense of node_modules brings only benefits, because you don't need to store many node_modules folders and decrease free space on a disk. Also, you won't encounter problems with IDE, because you don't work with those files directly.

So, I guess, yarn pnp allows you get all benefits without encountering its issues.

What are your thoughts? Am I missing any points?

**Comment by ije** [comment]
now pnpm works, i afraid changing to yarn may break the build system
